src/main/resources/org/csanchez/jenkins/plugins/kubernetes/NonConfigurableKubernetesCloud/config.jelly

```diff
@@ -1,5 +1,6 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:c="/lib/credentials">
+  <f:description>Cloud name : ${instance.displayName}</f:description>
   <f:invisibleEntry title="${%Cloud name}">
     <f:readOnlyTextbox name="cloudName" value="${instance.displayName}"/>
   </f:invisibleEntry>
```

Comment:
```suggestion
  <f:entry title="${%Cloud name}">
    <f:readOnlyTextbox name="cloudName" value="${instance.displayName}"/>
  </f:entry>
```
?

Reply:
That was my first patch actually but to be honest disabled textboxes cannot be well distinguished from enabled ones. Since this a global configuration page with editable parts and readonly ones, I think it's better from a UX perspective with a simple description.

![Capture d’écran 2022-08-29 à 10 43 43](https://user-images.githubusercontent.com/171459/187161925-036162a7-f3cf-4071-bf43-b426a1db0ccc.png)

vs. 

![Capture d’écran 2022-08-29 à 10 44 13](https://user-images.githubusercontent.com/171459/187161943-6473b741-dce8-437b-8aba-fdee6beaa66c.png)